packages/outline-form/src/outline-form.ts

```diff
@@ -12,9 +12,9 @@ import globalStyles from './outline-form.global.scoped.css.lit';
  */
 @customElement('outline-form')
 export class OutlineForm extends OutlineElement {
-  static styles: CSSResultGroup = [OutlineElement.styles, componentStyles];
-
-  lightDomStyles = new LightDomStyles(this, globalStyles);
+  static styles = componentStyles;
+  // Removing due to LightDomStyles dependency
+  //lightDomStyles = new LightDomStyles(this, globalStyles);
```

Comment:
Be aware that without the light dom styles, outline-form is going to break for any project using it (will no longer have proper styling)

Reply:
That is expected.

Reply:
After reviewing this carefully, the `outline-form` component is already broken: https://outline.phase2tech.com/?path=/docs/content-form--documentation

Something has changed through unlocked dependencies and a breaking change somewhere. 

Bottom line is this: 

- This is a blocker for new projects (and some current projects
- We have projects using the `lightDomStyles` controller from `@phase2/outline-core@0.1.12` (latest). 
- I can't remove that controller without it breaking this project if they update `outline-core`.